malib/algorithm/common/policy.py

```diff
@@ -76,6 +76,7 @@ def __init__(
             "use_cuda": False,
             "use_dueling": False,
             "preprocess_mode": Mode.FLATTEN,
+            "action_squash": False,
```

Comment:
The default `custom_config` should contain only the necessary parameters to avoid parameter cross-contamination between policies. Is the `action_squash` necessary for policy building?

Reply:
Move `action_squash` to SAC policy class.